nastaveni/nastaveni.php

```diff
@@ -180,3 +180,4 @@
 define('DEN_PRVNI_UBYTOVANI', DEN_PRVNI_DATE);                  // datum, kterému odpovídá ubytovani_den (tabulka shop_predmety) v hodnotě 0
 define('PROGRAM_ZACATEK', 8);   // první hodina programu
 define('PROGRAM_KONEC', 24);    // konec programu (tuto hodinu už se nehraje)
+define('SECRET_CRYPTO_KEY', 'def0000066cba9ae32fdda839a143276cc0646b3880920c93876ecc1bbaca96ee6ed251559516b1804f4742c2165e4c7eb3ed5c7a5abe857c6db8608e3b5fe97a8cdf15a');
```

Comment:
Hoď to do nastaveni/nastaveni-local-default.php (třeba za HTTPS_ONLY). Tam je to uzpůsobené, aby se dalo líp na produkci přepsat (tady by to dělalo problémy v gitu).